manifests/s/SnapmakerDevTeam/SnapmakerLuban/4.1.1/SnapmakerDevTeam.SnapmakerLuban.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: SnapmakerDevTeam.SnapmakerLuban
+PackageVersion: 4.1.1
+PackageLocale: en-US
+Publisher: Snapmaker Dev Team
+PublisherUrl: https://snapmaker.com/
+PublisherSupportUrl: https://snapmaker.com/support
```

```suggestion
#PublisherSupportUrl: https://snapmaker.com/support
```
ZenDesk URLs are blocked in Azure DevOps.

---

Ugh. Thats really annoying